Resources/Prototypes/Entities/Clothing/OuterClothing/bio.yml

```diff
@@ -21,6 +23,8 @@
     sprite: Clothing/OuterClothing/Bio/cmo.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Bio/cmo.rsi
+  - type: DiseaseProtection
+    protection: 0.3
```

Comment:
surely a bio suit can do better than that

Reply:
basic Bio suit + bio hood + gloves + mask will protect from basically everything except the weird alien artifacts (which you can push over with slightly better quality any of those 4 items

Reply:
in fact bio suit + bio hood alone will protect from everything but that

Reply:
aight